server/notification-providers/smspartner.js

```diff
@@ -0,0 +1,48 @@
+const NotificationProvider = require("./notification-provider");
+const axios = require("axios");
+
+class SMSPartner extends NotificationProvider {
+    name = "SMSPartner";
+
+    /**
+     * @inheritdoc
+     */
+    async send(notification, msg, monitorJSON = null, heartbeatJSON = null) {
+        const okMsg = "Sent Successfully.";
+        const url = "https://api.smspartner.fr/v1/send";
+
+        try {
+            //Lets remove non ascii char
+            let cleanMsg = msg.replace(/[^\x00-\x7F]/g, "");
+
+            let data = {
+                "apiKey": notification.smspartnerApikey,
+                "gamme": 1,
```

Comment:
what is `gamme`?

Reply:
This is the SMS range, its value must be:
1 for Premium SMS
2 for Low Cost SMS
If this parameter is omitted, SMS messages will be sent in the Premium range.
I think this can be removed.

Reply:
What is "premium" SMS? 🤔

Reply:
The "Premium" range is SMS that can be sent 24 hours a day, while eco can only be sent from 8am to 8pm. But SMS Partner no longer seems to make the ECO range. So the Premium range is induced by default.

Reply:
***ECO RANGE DOES WHAT???***

Would it not be better if we send in the eco range from 7:30am to 7:30am and in the premium range otherwise?

Reply:
It could have been a great idea! But unfortunately **eco** range no longer exists at SMS Partner :/ 
![Uploading CleanShot 2024-05-22 at 16.04.07@2x.png…]()

Reply:
It could have been a great idea! But unfortunately eco range no longer exists at SMS Partner :/
![CleanShot 2024-05-22 at 16 18 47@2x](https://github.com/louislam/uptime-kuma/assets/5034521/670586a4-c0a3-4cac-9dc0-a4ea3d70db5e)

Reply:
>  As of August 1, 2019, automated sending from 10-digit mobile numbers is no longer permitted on the platform (In accordance with Decision no. 2018-0881 of July 24, 2018).
“In order to strengthen users' confidence in the calls and messages they receive and to enable operators to better manage the flow of calls and SMS/MMS messages sent by automated systems, this decision provides that as of August 1, 2019, certain territorialized numbers (geographic, non-geographic and 10-digit mobile numbers) may no longer be used as caller IDs or message senders presented to the called party for calls or messages sent by automated systems.”

Reply:
So this parameter is a no-op? => then lets remove it, or am I missing something?

Reply:
Yes, I removed this param here : https://github.com/louislam/uptime-kuma/pull/4769/commits/f81cd4411627892eef60a48b6811f643dd97eebb

Sorry for the misunderstanding